### PR TITLE
[IMP] l10n_*: add demo company

### PR DESCRIPTION
**Pull request by william-andre**
Task [2198388](https://www.odoo.com/web#id=2198388&action=333&active_id=1691&model=project.task&view_type=form&cids=&menu_id=4720)

When testing and demoing localization, it can but cumbersome to create a Company with the right credentials and install the chart of account on it.
This commit aims to shorten the process. The first version only contains a valid vat (or similar) number, but we should add fields specific to localization in the future so that we have a working environment as soon as we install the localization. 

(created using https://gist.github.com/william-andre/cf687441e1bcd8cdaad7500fa1ee0d7a)

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ar/demo/demo_company.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (jco-odoo) [inline addons/l10n_ar/demo/demo_company.xml]: They have got 3 demo companies already.

File: addons/l10n_ae/demo/demo_company.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<odoo>
+    <record id="partner_demo_company_ae" model="res.partner">
+        <field name="name">AE Company</field>
+        
+        <field name="street">Street 123</field>
+        <field name="city">City</field>
+        <field name="country_id" ref="base.ae"/>
+        <field name="state_id" ref="base.state_ae_uq"/>
+        <field name="zip">2000</field>
+        <field name="phone">+1 555 123 8069</field>
```
Comment (jco-odoo) [inline addons/l10n_ae/demo/demo_company.xml]: Is there a way to use a bit realistic data?  We could use Odoo Dubai ?

Comment (william-andre) [reply addons/l10n_ae/demo/demo_company.xml]: The phone number is now well formatted, I will see if I find something not too complicated to generate more realistic addresses, but I don't think its really necessary. Maybe I can translate the Street name, and find the correct zip code format ?

**Comment by william-andre** [comment]
@jco-odoo I now have
* a valid VAT number (or similar depending on the locale)
* a well formatted phone number (with the actual local number being 1234...)
* a real address (I used an api to find a street (or shop, attraction, whatever) containing the letter 'a') in the country

**Comment by william-andre** [comment]
@jco-odoo Rebased, squashed, and i installed all l10n on a fresh db without any warning related to this task.
FYI here are the warnings I got; I can fix that in another branch if you need me to
```
2020-03-24 16:11:07,708 10712 WARNING master-l10n-demo-company-wan odoo.addons.base.models.ir_model: Two fields (l10n_latam_country_code, country_code) of account.move() have the same label: Country Code.
2020-03-24 16:11:09,241 10712 WARNING master-l10n-demo-company-wan odoo.modules.loading: The model l10n_co_reports.retention_report.wizard has no access rules, consider adding one. E.g. access_l10n_co_reports_retention_report_wizard,access_l10n_co_reports_retention_report_wizard,model_l10n_co_reports_retention_report_wizard,base.group_user,1,0,0,0
2020-03-24 16:11:09,241 10712 WARNING master-l10n-demo-company-wan odoo.modules.loading: The model l10n.sg.reports.iaf.wizard has no access rules, consider adding one. E.g. access_l10n_sg_reports_iaf_wizard,access_l10n_sg_reports_iaf_wizard,model_l10n_sg_reports_iaf_wizard,base.group_user,1,0,0,0
2020-03-24 16:11:09,241 10712 WARNING master-l10n-demo-company-wan odoo.modules.loading: The model l10n_uk.hmrc.send.wizard has no access rules, consider adding one. E.g. access_l10n_uk_hmrc_send_wizard,access_l10n_uk_hmrc_send_wizard,model_l10n_uk_hmrc_send_wizard,base.group_user,1,0,0,0
2020-03-24 16:11:09,241 10712 WARNING master-l10n-demo-company-wan odoo.modules.loading: The model f29.report.wizard has no access rules, consider adding one. E.g. access_f29_report_wizard,access_f29_report_wizard,model_f29_report_wizard,base.group_user,1,0,0,0
```

**Comment by rgo-odoo** [comment]
@william-andre FYI I have proposed fixes for these warnings for one of my tasks with ref #<a href="https://www.odoo.com/web#id=2212790&action=327&model=project.task&view_type=form&cids=2&menu_id=4720">2212790</a> in branch <a href="https://github.com/odoo/enterprise/compare/saas-13.2...odoo-dev:saas-13.2-nightly-l10n-fixes-rgo">saas-13.2-nightly-l10n-fixes-rgo</a>.

**Comment by AdrienHorgnies** [comment]
The following points might not be problems but I prefer to bring them to your attention:
-  Some countries have a VAT, some don't. I think that would be cool to have a valid VAT number for all demo companies. I frequently required to set one when I test localization stuff.
- Hongkong company is missing its address.

Other than that, I could install all localizations and didn't find any bug.

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**